docs/source/guide/ml.md

```diff
@@ -114,39 +175,35 @@ If you want to retrieve predictions manually for a list of tasks **using only an
   ]
 }
 ```
-### Enable auto-update for a model
-Enabling model auto-update allows automatic retraining of your model every time a new annotation is submitted. To enable auto-update, do the following:
-
-Either enable the **Allow version auto-update** option when adding a model, or:
-1. In the Label Studio UI, open the project that you want to use with your ML backend.
-2. Select **Settings > Machine Learning**.
-3. For the connected model you wish to enable, click **Edit** in the overflow menu.
-4. Enable **Allow version auto-update**.
-5. Click **Validate and Save**.
 
-### Get interactive pre-annotations
+### Interactive pre-annotations
 
 ML-assisted labeling with interactive pre-annotations works with image segmentation and object detection tasks using rectangles, ellipses, polygons, brush masks, and keypoints, as well as with HTML and text named entity recognition tasks. Your ML backend must support the type of labeling that you're performing, recognize the input that you create, and be able to respond with the relevant output for a prediction.
 
-Either enable the **Use for interactive preannotation** option when adding a model, or:
+Either enable the **Interactive preannotations** option when adding a model, or use the **Edit** action from the project settings page to enable/disable this option. 
 
-1. In the Label Studio UI, open the project that you want to use with your ML backend.
-2. Select **Settings > Machine Learning**.
-3. For the connected model you wish to enable, click **Edit** in the overflow menu.
-4. Enable **Use for interactive preannotation**.
-5. Click **Validate and Save**.
+#### Smart tools
 
-For image labeling, you can update your labeling configuration to include `smart="true"` option for the type of labeling you're performing. Smart tools appear by default if auto-annotation is enabled. If you only want the smart option to appear and don't want to perform manual labeling at all, use `smartOnly="true"`. 
+Smart tools duplicate the tools that you have set up to use through your labeling configuration (e.g. rectangles) to allow interaction with the ML backend. Smart tools are currently for Rectangle, Ellipse, Polygon, Keypoint, and Brush tags. 
```

Comment:
shall we provide exact labeling tags instead?

Reply:
Added tag formatting and links to the appropriate tag pages